contrib/core/actions/send_mail/send_mail

```diff
@@ -8,9 +8,12 @@ TO=$1
 shift
 SUBJECT=$1
 shift
+SEND_EMPTY_BODY=$1
+shift
 BODY="$@"
 
-${MAIL} <<EOF
+if [[ -z "${BODY// }" && $SEND_EMPTY_BODY == 'True' ]] || [[ -n $BODY ]]; then
```

Comment:
My only thought right away ... what is the literal `true` being sent? `True`, `true` and a ton other ways to express `true` as a bareword are valid in YAML. Will this catch at least `true` and `True`?